src/Idris/Parser.idr

```diff
@@ -1306,7 +1306,7 @@ logLevel fname
 directive : OriginDesc -> IndentInfo -> Rule Directive
 directive fname indents
     = do decoratedPragma fname "hide"
-         n <- name
+         n <- (fixityNS <|> name)
```

Comment:
This really feels like a hack: why not have a
```idris
data HidingDirective = HideName Name | HideFixity Fixity Name
```
and have `Hide` take a `HidingDirective` rather than a `Name`
instead of manufacturing an illegal namespace in `fixityNS` and then looking for it
in `isOperatorFixityNS`?

Reply:
Should be fixed